eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/HttpResponseUtils.java

```diff
@@ -29,37 +27,38 @@
 import io.netty.handler.codec.http.HttpResponse;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.HttpVersion;
+import org.apache.eventmesh.common.Constants;
```

Comment:
please don't change this import order

Reply:
I think it must optimize

Reply:
but this not followed our project checkstyle config, the ci check won't be succeeded.

Reply:
I think eventmesh need optimize or refactor some nonstandard code.

Reply:
I'm not clearly get what you mean, why you change the package import order ?